File: src/main/java/top/criwits/scaleda/verilog/psi/factory/nodes/SimpleIdentifierPsiLeafNode.kt
package top.criwits.scaleda.verilog.psi.factory.nodes

import org.antlr.intellij.adaptor.psi.ANTLRPsiLeafNode
import top.criwits.scaleda.verilog.parser.VerilogLexer
import top.criwits.scaleda.verilog.psi.factory.VerilogPsiLeafNodeFactory

class SimpleIdentifierPsiLeafNode(text: CharSequence) :
    ANTLRPsiLeafNode(VerilogPsiLeafNodeFactory.getTokenIElementType(VerilogLexer.Simple_identifier), text) {

    override fun getName(): String {
        return this.text
    }

}

File: src/main/scala/top/criwits/scaleda/verilog/psi/nodes/SimpleIdentifierPsiLeafNode.scala
package top.criwits.scaleda
package verilog.psi.nodes

import org.antlr.intellij.adaptor.psi.ANTLRPsiLeafNode
import top.criwits.scaleda.verilog.parser.VerilogLexer
import top.criwits.scaleda.verilog.psi.VerilogPsiLeafNodeFactory

class SimpleIdentifierPsiLeafNode(text: CharSequence)
    extends ANTLRPsiLeafNode(VerilogPsiLeafNodeFactory.getTokenIElementType(VerilogLexer.Simple_identifier), text) {
  override def getName: String = getText
}
